go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/google/go-containerregistry v0.19.1
 	github.com/google/go-querystring v1.1.0
-	github.com/google/go-tpm-tools v0.4.3
+	github.com/google/go-tpm-tools v0.4.4
```

Comment:
Ye olde patch release with breaking changes: https://github.com/google/go-tpm-tools/releases/tag/v0.4.4.

Seems like most of these is stuff that people shouldn't be using anyway, so I'm assuming we are unaffected.